### ISSUE TITLE
proposal: go tool pprof should show tooltip info at flame tips

### ISSUE THREAD

**Issue by ORESoftware** [golang/go#66238]
### Proposal Details

For example:

![image](https://github.com/golang/go/assets/11139560/b8c250f0-ca60-48d7-a28d-79828a66adf1)

and then zooming in on the right:

![image](https://github.com/golang/go/assets/11139560/192bf650-118f-415b-8457-3d06231f69b6)

I would assume if I hove over a thin section, I can get a tooltip message about what it is - but in Chrome at least, no such tooltip message.

In other words, looking for tooltip info on tips of flames

**Comment by seankhliao** [comment]
this is an upstream pprof issue
